bugbug/db.py

```diff
@@ -23,20 +23,24 @@ def register(path, url):
 
 # Download and extract databases.
 def download():
-    for path, url in DATABASES.items():
-        if os.path.exists(path):
+    for path, info in DATABASES.items():
+        if os.path.exists(path) and not is_outdated(path):
             continue
 
         xz_path = f'{path}.xz'
 
+        ver_path = 'data/DB_VERSION.json'
```

Comment:
Define this as a constant at the top of the file, and use it both in `is_outdated` and `update_ver_file`